install/installer/pkg/components/ide-service/ide_config_configmap.go

```diff
@@ -49,7 +49,7 @@ func ideConfigConfigmap(ctx *common.RenderContext) ([]runtime.Object, error) {
 	}
 
 	jbPluginImage := ctx.ImageName(ctx.Config.Repository, ide.JetBrainsBackendPluginImage, ctx.VersionManifest.Components.Workspace.DesktopIdeImages.JetBrainsBackendPluginImage.Version)
-	jbPluginLatestImage := ctx.ImageName(ctx.Config.Repository, ide.JetBrainsBackendPluginImage, ctx.VersionManifest.Components.Workspace.DesktopIdeImages.JetBrainsBackendPluginLatestImage.Version)
+	jbPluginLatestImage := ctx.ImageName(ctx.Config.Repository, ide.JetBrainsBackendPluginImage, "latest")
```

Comment:
I believe that for preview envs you want to have changes from your branch, not from docker. How else you would test latest version of plugin?

Reply:
@akosyakov that's a good point. I was only thinking about SH. For SH do we agree that it also makes sense to have backend-plugin to actual `latest` instead of pinned to a commit?

Reply:
yes, it is necessary

Reply:
Updated